setup.cfg

```diff
@@ -16,19 +17,30 @@ DJANGO_SETTINGS_MODULE = kaavapino.settings
 norecursedirs = bower_components node_modules .git .idea
 doctest_optionflags = NORMALIZE_WHITESPACE IGNORE_EXCEPTION_DETAIL ALLOW_UNICODE
 
-[isort]
-atomic=true
-combine_as_imports=false
-indent=4
-known_standard_library=token,tokenize,enum,importlib
-known_third_party=django,six
-length_sort=false
-line_length=120
-multi_line_output=5
-not_skip=__init__.py
-order_by_type=false
-skip=migrations,node_modules
-wrap_length=120
-
 [pydocstyle]
 ignore=D100,D104,D105,D200,D203,D400
+
+[prequ]
+annotate = yes
+requirements =
+    Django~=2.1
+    django-helusers
+    django-environ==0.4.4
+    django-stronghold
+    psycopg2-binary
```

Comment:
I read from [psycopg2 PyPi](https://pypi.org/project/psycopg2/) that: _The binary package is a practical choice for development and testing but in production it is advised to use the package built from sources._

From [official article](http://initd.org/psycopg/articles/2018/02/08/psycopg-274-released/) I understood that `psycopg2-binary` is not necessarily deprecating `psycopg2`, but from 2.8 onwards \*-binary will be a wheels-only build and psycopg2 will build the library from source (requires a C compiler and of Python/Postgres dev packages at build time, and libpq at runtime).

Binary is of course more convenient to install. Just thinking if we should force psycopg2 package to be built from source.

This was just a thought, I think we can go with the binary package (for now at least).

Reply:
You are probably right here. I'll revert back.